tests/common/mock_cgroup_commands.py

```diff
@@ -1,206 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
This file was renamed to mock_cgroup_environment.py. In the _MOCKED* arrays stay the same in the new file and the mocking logic is now moved to mock_environment.py